src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,673 @@
+        >
+          Registration type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Register the system later
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const RegisterNowList = () => {
+  const activationKey = useAppSelector((state) => selectActivationKey(state));
+  const { isError } = useShowActivationKeyQuery(
+    // @ts-ignore - type of 'activationKey' might not be strictly compatible with the expected type for 'name'.
+    { name: activationKey },
+    {
+      skip: !activationKey,
+    }
+  );
+  return (
+    <>
+      <TextContent>
+        <TextList component={TextListVariants.dl}>
+          <TextListItem
+            component={TextListItemVariants.dt}
+            className="pf-u-min-width"
+          >
+            Registration type
+          </TextListItem>
+          <TextListItem
+            component={TextListItemVariants.dd}
+            data-testid="review-registration"
+          >
+            {/*   <TextList isPlain>
+              {getState()?.values?.['register-system']?.startsWith(
+                'register-now'
+              ) && (
+                <TextListItem>
+                  Register with Red Hat Subscription Manager (RHSM)
+                  <br />
+                </TextListItem>
+              )}
+              {(getState()?.values?.['register-system'] ===
+                'register-now-insights' ||
+                getState()?.values?.['register-system'] ===
+                  'register-now-rhc') && (
+                <TextListItem>
+                  Connect to Red Hat Insights
+                  <br />
+                </TextListItem>
+              )}
+              {getState()?.values?.['register-system'] ===
+                'register-now-rhc' && (
+                <TextListItem>
+                  Use remote host configuration (rhc) utility
+                  <br />
+                </TextListItem>
+              )}
+            </TextList> {*/}
+          </TextListItem>
+          <TextListItem component={TextListItemVariants.dt}>
+            Activation key
+            <Popover
+              bodyContent={
+                <TextContent>
+                  <Text>
+                    Activation keys enable you to register a system with
+                    appropriate subscriptions, system purpose, and repositories
+                    attached.
+                    <br />
+                    <br />
+                    If using an activation key with command line registration,
+                    you must provide your organization&apos;s ID. Your
+                    organization&apos;s ID is{' '}
+                    {/*    {getState()?.values?.['subscription-organization-id'] !==
+                    undefined ? (
+                      getState()?.values?.['subscription-organization-id']
+                    ) : (
+                      <Spinner size="md" />
+                    )}*/}
```

Comment:
We don't store the organization-id anywhere yet. Either we could get it via an API request or we can also add it to the store and initialize it on the Registration step if needed.

Reply:
what do you think is better one?